PVList.md

```diff
@@ -0,0 +1,354 @@
+# IPMI Manager Process Variables naming list
+
+This document have the purpose of making easier to understand the substitution
+tree on this IOC and to check if the [LNLS's naming convention](https://wiki-sirius.lnls.br/mediawiki/index.php/Machine:Naming_System)
+of is being followed. It is also useful for any user to check what PVs are
+available.
+Each new record created or aliased from the ipmiComm module must be documented
+here.
+
+```
+AREA=IA-{{01..20}RaBPM|20RaBPMTL} # Area = LNLS uTCA crates
+DIS=CO  #Discipline = Control
+```
+# MCH
```

Comment:
You should increase the header level of all following headers by one, so that the title becomes semantically the top-level header.